src/search-modal/SearchFilterWidget.jsx

```diff
@@ -20,12 +20,22 @@ import {
  * When clicked, the button will display a dropdown menu containing this
  * element's `children`. So use this to wrap a <RefinementList> etc.
  *
- * @type {React.FC<{appliedFilters: {label: React.ReactNode}[], label: React.ReactNode, children: React.ReactNode}>}
+ * @type {React.FC<{
+ *   appliedFilters: {label: React.ReactNode}[],
+ *   label: React.ReactNode,
+ *   children: React.ReactNode,
+ *   filterType?: string
+ * }>}
  */
 const SearchFilterWidget = ({ appliedFilters, ...props }) => {
   const [isOpen, open, close] = useToggle(false);
   const [target, setTarget] = React.useState(null);
 
+  const iconMap = {
+    blockType: FilterList,
+    tags: Tag,
+  };
```

Comment:
The `SearchFilterWidget` is a generic component, and it shouldn't need to know about what type of filters may use it. In other words, it shouldn't need `iconMap`. Instead, I recommend it just accept an `icon` prop, and then the widgets like `<FilterByBlockType>` can pass in `icon={FilterList}`.

Reply:
Good point, thanks for that! I've updated it